src/skibidi_orm/migration_engine/adapters/sqlite3_adapter.py

```diff
@@ -1,26 +1,34 @@
 from __future__ import annotations
-from typing import Literal
-from skibidi_orm.migration_engine.adapters.base_adapter import (
-    BaseAdapter,
-    BaseColumn,
-    Relation,
-    BaseTable,
-)
-from skibidi_orm.migration_engine.operations.constraints import Constraint
+
+from skibidi_orm.migration_engine.adapters.base_adapter import BaseAdapter
+from skibidi_orm.migration_engine.adapters.sqlite3_typing import SQLite3Typing
+from skibidi_orm.migration_engine.db_inspectors.sqlite3_inspector import SqliteInspector
+
+from skibidi_orm.migration_engine.schema_analysys.state_manager import StateManager
+
+from skibidi_orm.migration_engine.operations.table_operations import TableOperation
+from skibidi_orm.migration_engine.operations.column_operations import ColumnOperation
 
 
 class SQLite3Adapter(BaseAdapter):
 
-    DataTypes = Literal["TEXT", "INTEGER", "REAL", "BLOB", "NULL"]
-    Constraints = Constraint
-    Column = BaseColumn[DataTypes]
-    Table = BaseTable[Column]
-    Relation = Relation
+    DataTypes = SQLite3Typing.DataTypes
+    Constraints = SQLite3Typing.Constraints
+    Column = SQLite3Typing.Column
+    Table = SQLite3Typing.Table
+    Relation = SQLite3Typing.Relation
```

Comment:
Maybe it's not a good idea to add new class only to store types, since we would need to add new class for each supported database engine instead of creating just new type alias.
For example Relation field will be the the same for every supported engine